src/transform_astronomy_data.py

```diff
@@ -0,0 +1,63 @@
+'''Gets JSON data from specified file, filters for relevant columns and converts into dataframe'''
+
+# pylint:disable=line-too-long
+
+import json
+import pandas as pd
+
+FILE_PATH = "data/planetary_data.json"
+
+
+def get_json_data(file_path: str) -> dict:
+    '''Gets JSON from file and returns'''
+    try:
+        with open(file_path, 'r', encoding='utf-8') as file:
+            return json.load(file)
+    except FileNotFoundError:
+        print('Error, No file found')
+        return None
```

Comment:
suggestion: to align with type hints, return an empty dictionary instead of `None` or raise an error instead
```suggestion
        return {}
```